pkg/services/sqlstore/annotations_cleanup.go

```diff
@@ -0,0 +1,49 @@
+package sqlstore
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/grafana/grafana/pkg/bus"
+	m "github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/util/errutil"
+)
+
+func init() {
+	bus.AddHandler("sql", deleteExpiredAnnotations)
+}
+
+const MAX_EXPIRED_ANNOTATIONS_TO_DELETE = 900
+
+func deleteExpiredAnnotations(cmd *m.DeleteExpiredAnnotationsCommand) error {
+	return inTransaction(func(sess *DBSession) error {
```

Comment:
Return this as `inTransaction` performs all these SQL operations in one transaction. I dont think that's necessary. You can use `withDbSession` instead to avoid transactions.